packages/@ember/-internals/runtime/lib/system/core_object.js

```diff
@@ -111,7 +111,7 @@ function initialize(obj, properties) {
       }
 
       if (isDescriptor) {
-        possibleDesc.set(obj, keyName, value);
+        obj[keyName] = value;
```

Comment:
Chatted with @pzuraq about this, he is going to bring back the prior code (to avoid looking up the descriptor _again_).